app/routes/tags/index.js

```diff
@@ -1,16 +1,7 @@
 import AuthenticatedRoute from 'ghost-admin/routes/authenticated';
-import {inject as service} from '@ember/service';
 
 export default AuthenticatedRoute.extend({
-    mediaQueries: service(),
-
     beforeModel() {
-        let firstTag = this.modelFor('tags').get('firstObject');
-
         this._super(...arguments);
-
-        if (firstTag && !this.get('mediaQueries.maxWidth600')) {
-            this.transitionTo('tags.tag', firstTag);
-        }
     }
 });
```

Comment:
It seems like this file can just be removed at this state as default route will kickin, but kept it for now till the final changes are complete 😬

Reply:
The empty `beforeModel()` can be removed but we still want this to be an `AuthenticatedRoute` instance rather than the default auto-generated `Route`

Reply:
Ignore the above, missed that this was the index and not the `tags` route 🙈 The file can be removed 👍